src/styles/Card.js

```diff
@@ -110,6 +111,24 @@ const ContextStyle = styled(Card)`
   a {
     color: ${colors.lighterGrey};
   }
+
+  .languages {
+    margin-top: 20px;
+    
+    .dot {
+      font-size: 25px;
+      margin: 0;
+      vertical-align: sub;
+    }
+    
+    .name {
+      color: black;
```

Comment:
Define color as a variable in "./variables.js" file and include here, eg. - 
${colors.black}

Reply:
This is a good call. I have not implemented Dark mode before but it may make it easier to implement with all colors leveraging variables. #607 

Thoughts @filiptronicek?

Reply:
@bdougie I agree, it will be much easier to do it if the colors will be stored in vars.